web/includes/auth.php

```diff
@@ -60,7 +60,12 @@ function migrateHash($user, $pass) {
 function validateUser($username='', $password='') {
   $sql = 'SELECT * FROM Users WHERE Enabled=1 AND Username=?';
   // local user, shouldn't affect the global user
-  $user = dbFetchOne($sql, NULL, array($username));
+  $user = null; // Not global
+  if (ZM_CASE_INSENSITIVE_USERNAMES) {
+      $user = dbFetchOne($sql, NULL, array(strtolower($username)));
+  } else {
+      $user = dbFetchOne($sql, NULL, array($username));
+  }
```

Comment:
This requires the value stored in the db to be lowercase as well.  I think we should alter the sql as well to use tolower(Username).